clirr-ignores.xml

```diff
@@ -44,6 +44,12 @@
         <justification>This class is only present if the project was compiled with JDK 8+</justification>
     </difference>
 
+    <difference>
```

Comment:
This seems a bit odd, you shouldn't see Clirr warnings when you add new classes. I don't think you need these exceptions.

Reply:
I'll get rid of those changes.